ccmlib/scylla_node.py

```diff
@@ -216,6 +216,10 @@ def get_tool_args(self, toolname):
         raise NotImplementedError('ScyllaNode.get_tool_args')
 
     def get_env(self):
+        try:
+            return self._launch_env
+        except AttributeError:
```

Comment:
Could we fix the assignment of this variable so it is null instead of missing instead? Catching AttributeError is very ugly

Reply:
Done in v2.
@pehala Please re-review